From 633172a2c1db9712a7a3f924e7561f8b42b15669 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 7 Jan 2015 18:12:35 +0800
Subject: [PATCH] MLK-10244-4 ARM: dts: imx7d: enable sdma driver by default

enable sdma driver by default.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 89c0b6d46f1fc79e81679f5aa2c2a457efc078cf)
---
 arch/arm/boot/dts/imx7d.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 4b97e57..e63412a 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -869,7 +869,6 @@
 				clock-names = "ipg", "ahb";
 				#dma-cells = <3>;
 				fsl,sdma-ram-script-name = "imx/sdma/sdma-imx7d.bin";
-				status = "disabled";
 			};
 
 			fec1: ethernet@30be0000 {
-- 
1.8.0

